src/Sulu/Bundle/SearchBundle/DependencyInjection/Configuration.php

```diff
@@ -32,11 +32,18 @@ public function getConfigTreeBuilder()
                             ->scalarNode('security_context')->end()
                             ->arrayNode('contexts')
                                 ->prototype('scalar')
-                                ->defaultValue([])
+                                    ->defaultValue([])
+                                ->end()
                             ->end()
                         ->end()
                     ->end()
                 ->end()
+            ->end()
+            ->children()
+                ->arrayNode('website_indexes')
+                    ->prototype('scalar')->end()
+                    ->defaultValue(['page_{{ webspace.key }}_published'])
```

Comment:
I am not sure if this is the right place... Could imagine this should be going into the webspace config, because different webspaces might or might not want to include certain indexes... That would then also make the replacer in here obsolete.

As a default I would still use the same index as currently.